docs/engineering/app/datastore/tagging.mdx

```diff
@@ -19,7 +19,7 @@ System Suffix|Display|Purpose
 `practitioner-tag-id`|Practitioner|This is the Practitioner that is logged into the app when the resource is created.
 `location-tag-id`|Practitioner Location|This is the Location linked to the Organization of the Practitioner that is logged into the app when the resource is created.
 `organisation-tag-id`|Practitioner Organization|This is the Organization linked to the Practitioner that is logged into the app when the resource is created.
-`related-entity-location-tag-id`|Related Entity Location|"Entity" here is a `Patient`, `Group`, Point of Service (as a `Location` resource), or other organizing unit, and this stores the ID of a `Location` resource (or the resource itself if it is a `Location`) lnked to that entity.
+`related-entity-location-tag-id`|Related Entity Location|"Entity" here is a `Patient`, `Group`, Point of Service (as a `Location` resource), or other organizing unit, and this stores the ID of a `Location` resource (or the resource itself if it is a `Location`) linked to that entity.
```

Comment:
those "organizing unit" in this context mean anything? Like it could be any resource?

Reply:
or any resource that can be connected to a location?